README.md

```diff
@@ -150,61 +150,61 @@ You can use the following parameters in prometheus configuration file. The `para
 
 ## Current Status
 
-- VCenter and ESXi 6 and 6.5 have been tested.
+- vCenter and vSphere 6.0/6.5 have been tested.
 - VM information, Snapshot, Host and Datastore basic information is exported, i.e:
 ```
-# HELP vmware_snapshots VMWare current number of existing snapshots
+# HELP vmware_snapshots VMware current number of existing snapshots
 # TYPE vmware_snapshot_count gauge
 vmware_snapshot_timestamp_seconds{vm_name="My Super Virtual Machine"} 2.0
-# HELP vmware_snapshot_timestamp_seconds VMWare Snapshot creation time in seconds
+# HELP vmware_snapshot_timestamp_seconds VMware Snapshot creation time in seconds
 # TYPE vmware_snapshot_timestamp_seconds gauge
 vmware_snapshot_age{vm_name="My Super Virtual Machine",vm_snapshot_name="Very old snaphot"} 1478146956.96092
 vmware_snapshot_age{vm_name="My Super Virtual Machine",vm_snapshot_name="Old snapshot"} 1478470046.975632
 
-# HELP vmware_datastore_capacity_size VMWare Datasore capacity in bytes
+# HELP vmware_datastore_capacity_size VMware Datastore capacity in bytes
 # TYPE vmware_datastore_capacity_size gauge
 vmware_datastore_capacity_size{ds_name="ESX1-LOCAL"} 67377299456.0
-# HELP vmware_datastore_freespace_size VMWare Datastore freespace in bytes
+# HELP vmware_datastore_freespace_size VMware Datastore freespace in bytes
 # TYPE vmware_datastore_freespace_size gauge
 vmware_datastore_freespace_size{ds_name="ESX1-LOCAL"} 66349694976.0
-# HELP vmware_datastore_uncommited_size VMWare Datastore uncommitted in bytes
+# HELP vmware_datastore_uncommited_size VMware Datastore uncommitted in bytes
 # TYPE vmware_datastore_uncommited_size gauge
 vmware_datastore_uncommited_size{ds_name="ESX1-LOCAL"} 0.0
-# HELP vmware_datastore_provisoned_size VMWare Datastore provisoned in bytes
+# HELP vmware_datastore_provisoned_size VMware Datastore provisoned in bytes
 # TYPE vmware_datastore_provisoned_size gauge
 vmware_datastore_provisoned_size{ds_name="ESX1-LOCAL"} 1027604480.0
-# HELP vmware_datastore_hosts VMWare Hosts number using this datastore
+# HELP vmware_datastore_hosts VMware Hosts number using this datastore
 # TYPE vmware_datastore_hosts gauge
 vmware_datastore_hosts{ds_name="ESX1-LOCAL"} 1.0
-# HELP vmware_datastore_vms VMWare Virtual Machines number using this datastore
+# HELP vmware_datastore_vms VMware Virtual Machines number using this datastore
 # TYPE vmware_datastore_vms gauge
 vmware_datastore_vms{ds_name="ESX1-LOCAL"} 0.0
 
-# HELP vmware_host_power_state VMWare Host Power state (On / Off)
+# HELP vmware_host_power_state VMware Host Power state (On / Off)
 # TYPE vmware_host_power_state gauge
 vmware_host_power_state{host_name="esx1.company.com"} 1.0
-# HELP vmware_host_cpu_usage VMWare Host CPU usage in Mhz
+# HELP vmware_host_cpu_usage VMware Host CPU usage in MHz
```

Comment:
just nit-picking here, but correct abbreviation for megahertz is actually `MHz` (following: Hz, kHz, MHz, GHz notations in electronics)

Reply:
ignore me... looks like I need to check my vision, but the z is indeed lower-case 🤦‍♂️